packages/terra-paginator/src/_PaginatorButton.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { disableFocusActiveStyles, disableFocusOnBlur, enableFocusActiveStyles } from './_paginationUtils';
+import styles from './Paginator.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Sets aria-current attribute.
+   */
+  ariaCurrent: PropTypes.oneOfType([
```

Comment:
I might be missing it, but it seems this prop is only used 2x in the changes and both are passing a boolean value. When would a string value be passed?

Reply:
Node evaluates `1 === 1 && 'page'` as a string but, as best as I can tell, there isn't a reason to pass in the string 'page' to ariaCurrent and it just makes the logic confused, so I took it out and fixed the prop type, and it still functions as expected -  [3c4f04a](https://github.com/cerner/terra-core/pull/2686/commits/3c4f04a82bd80d77dc4fc4f35ba36c5d85deb003)